source/write-to-mongodb.txt

```diff
@@ -28,6 +28,18 @@ Write to MongoDB
 
          .. include:: /scala/write-to-mongodb.txt
 
+.. warning::
+
+   If you specify the ``overwrite`` save mode, the connector drops the target
+   collection and creates a new collection that uses the
+   default collection options.
+   This can affect collections that don't use the default options,
+   including the following collection types:
```

Comment:
comment: since the clause starting with "including" follows the noun "options," this reads a little confusing. 

A possible change would be to break the sentence into two, but leaving it up to you to decide if changes are needed (as this is a nit)